src/app/src/main/java/com/example/qr_go/Owner.java

```diff
@@ -20,6 +20,14 @@ public Owner()   {
         super();
     }
 
+    /**
+     * Constructor for owner class for mock data
+     */
+    @RequiresApi(api = Build.VERSION_CODES.O)
+    public Owner(String uuid, String password, String username, Integer totalScore, String email) {

```

Comment:
Good point about the constructor. This is fixed.

Concerning the `totalScore` variable and the `getTotalScore()` method, that was the original plan, but I had some concerns while working with firestore yesterday.

1. It's kinda slow. Might not be a good idea to have to perform the query for every user during searches/sorts in the SearchActivity.
2. There is no way to reliably return values of the query. Any data you get from the query has to be used in the listener as the queries are done concurrently. An alternative I tried was initializing a variable before the query, use the query to get my values, and return it after the query. Unfortunately, because the query is asynchronous and slow, it only ever really returns a null object (unless you're debugging).